### DISCUSSION TITLE
[Bug] Checkout - address change - state dropdown

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#414] [Help] [answered]
Ahoi. If one lands on the checkout page already logged in and then wants to change an address, the `state` select ("Kanton" in our screenshot) is shown with the US states filled in. But country Switzerland is default and selected which has no states assigned by default. Saving also doesn't work and would lead to wrong data sets.

At creating a new address, this is working properly.

![Screenshot 2023-10-24 at 15-42-34 Checkout](https://github.com/swellstores/community/assets/48205514/f1b74b63-26f8-4c8e-bbb6-f71e24b03943)

**Comment by sags95** [comment, marked as answer]
@josias-hosang Thank you reporting this, I'll be able to file this with the team so we can address it.
